src/Aspire.Hosting.Azure.Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -302,4 +279,93 @@ private static async Task<bool> ExecuteCommand(ProcessSpec processSpec)
 
         return null;
     }
+
+    internal static string GetChecksum(AzureBicepResource resource, JsonObject parameters)
+    {
+        // TODO: PERF Inefficient
+
+        // Combine the parameter values with the bicep template to create a unique value
+        var input = parameters.ToJsonString() + resource.GetBicepTemplateString();
+
+        // Hash the contents
+        var hashedContents = Crc32.Hash(Encoding.UTF8.GetBytes(input));
```

Comment:
Crc is so retro - can't use GetHashCode because it's cross process lifetime, I guess? There must be something that can take the string directly.

Reply:
Like it's the 80s